core/app/modules/import-export/assets/js/shared/dashboard-button/dashboard-button.js

```diff
@@ -0,0 +1,16 @@
+import Button from 'elementor-app/ui/molecules/button';
+
+import useLink from '../../hooks/use-link/use-link';
+
+export default function DashboardButton() {
```

Comment:
Isn't it supposed to be a general component

Reply:
Done.